README.md

```diff
@@ -1,16 +1,32 @@
 # lone-wolf-dark README
 
-## Working with Markdown
+## Installation
 
-You can author your README using Visual Studio Code. Here are some useful editor keyboard shortcuts:
 
-* Split the editor (`Cmd+\` on macOS or `Ctrl+\` on Windows and Linux).
-* Toggle preview (`Shift+Cmd+V` on macOS or `Shift+Ctrl+V` on Windows and Linux).
-* Press `Ctrl+Space` (Windows, Linux, macOS) to see a list of Markdown snippets.
+### Install By Searching
 
-## For more information
+1.  Install [Visual Studio Code](https://code.visualstudio.com/)
+2.  Launch Visual Studio Code
+3.  Choose **Extensions** from menu
+4.  Search for `lone-wolf-dark`
+5.  Click **Install** to install it
+6.  Click **Reload** to reload the Code
```

Comment:
I don't think these steps need to be added in readme because if someone has reached to this extension than
they are already in `Extensions` panel in vscode

Reply:
![image](https://user-images.githubusercontent.com/83064473/192097428-2b0679c2-b9b6-44f2-a42c-c6d51a236845.png)
 Hi aayushchugh,
If u can see then if we go to from marketplace of visual studio then there readme is used so we can add that if its is there
 then its okay cause user can choose any way to install extensions.

its okay if its there but  yeah need to remove point no 6  reload will be done soon.
Thanks